### PR TITLE
add post for distributions support

### PR DESCRIPTION
**Pull request by jburel**
Support plan

### CODE REVIEW

**Comment by jburel** [comment]
Available at https://snoopycrimecop.github.io/www.openmicroscopy.org/2023/07/24/linux-distributions.html

File: _posts/2023-07-24-linux-distributions.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+We are working with the Glencoe software team to test and release a series of changes across the OMERO components to workaround several of the known CentOS 7 issues and maintain compatibility for the upcoming 6 months. Once support has ended, these changes will be removed.
+
+We will update our documentation and update the OMERO components to work with Rocky 9 and Ubuntu 22.04 LTS.
+
+## Can I still install OMERO on CentOS 7/RHEL7?
+The OME team will no longer install new deployments on CentOS 7/RHEL 7 and will no longer test the daily builds on CentOS 7/RHEL 7.
+
+## How should I upgrade?
+
+OME supports the installation of OMERO over a range of supported platforms.
```
Comment (pwalczysko) [inline _posts/2023-07-24-linux-distributions.md:58]: Actually now I do not understand this. Does that mean range == "Rocky 9" & "Ubuntu 22.04" == 2 platforms ?

**Comment by pwalczysko** [comment]
Lgtm, except maybe we should reformulate https://github.com/ome/www.openmicroscopy.org/pull/658#discussion_r1272011394 - it sounds now like "you can upgrade to a range of platforms..." but in fact it is "you can upgrade to either Rocky 9 or Ubuntu 22.04". Maybe I am not getting it right ?

File: _posts/2023-07-24-linux-distributions.md (hunk 1 above)
Comment (jburel) [reply _posts/2023-07-24-linux-distributions.md]: adjusted accordingly

File: _posts/2023-07-24-linux-distributions.md [hunk 2]
```diff
@@ -55,7 +55,8 @@ The OME team will no longer install new deployments on CentOS 7/RHEL 7 and will
 
 ## How should I upgrade?
 
-OME supports the installation of OMERO over a range of supported platforms.
+OME will provide installation instructions and will test the installation on two selected platforms
+i.e. Rocky 9  and Ubuntu 22.04 LTS.
```
Comment (pwalczysko) [inline _posts/2023-07-24-linux-distributions.md:59]:
```suggestion
i.e. Rocky 9 and Ubuntu 22.04 LTS.
```

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**